### DISCUSSION TITLE
Retirar expressão “preferência sexual” do formulário de submissão de palestras para a Python Brasil

### DISCUSSION THREAD

**Discussion by cuducos** [apyb/comunidade#30] [Geral]
Num mesmo parágrafo a organização da Python Brasil diz “respeitar a diversidade e utiliza uma expressão agressiva para a comunidade LGBTQI+: “preferência sexual”.

Se a organização respeita, deve utilizar outra expressão.
Se não respeita, deve parar de dizer que respeita.

[Preferência sexual não é mesma coisa que identidade de gênero](https://www.glaad.org/how-sexual-orientation-different-gender-identity), e é uma expressão normalmente utilizada em [cenários de violência contra a comunidade](https://esqrever.com/2020/10/27/porque-o-termo-preferencia-sexual-esta-desatualizado-e-e-ofensivo/) (por exemplo, em terapias de conversão).

**Comment by rougeth** [comment]
Obrigado @cuducos, enviei essa mensagem para o Discord da organização.

**Comment by renan-asantos** [comment]
Corrigido @cuducos, valeu pelo aviso!

**Reply by gabubellon** [reply]
@renan-assis-santos achou que faltou ajustar o em ingles

**Reply by renan-asantos** [reply]
Opa, ajustado o inglês tbm, valeu @gabubellon

**Comment by gabubellon** [comment]
🏳️‍🌈  Nice catch @cuducos !!!

**Comment by gabubellon** [comment]
E obrigado pela rápida resposta diretoria e organização

**Reply by cuducos** [reply]
Sim, valeu demais, @renan-assis-santos 💜

**Comment by anadulce** [comment]
@cuducos estamos planejando fazer um post sobre o perfil sociocultural das pessoas proponentes.
Você considera aceitável o termo "orientação sexual"? Se não, tem alguma outra recomendação?
Muito obrigada!!

**Reply by cuducos** [reply]
Depende um pouco do que vocês querem com o recorte, sabe?

Pegando uma parte de um dos links que coloquei ali para cima:

> Identidade de gênero é o seu próprio, interno senso de ser uma homem ou uma mulher (ou como alguém fora does gêneros binários).
>
> Orientação sexual descreve a atração estável, física romântica e/ou emocional por outra pessoa (por exemplo: hétero, gay, lésbica, bissexual).
>
> Uma pessoa transgênera pode ser hétero hétera, gay ou lésbica.

Não sei se você perguntou apenas no contexto de _orientação sexual_ ao invés de _preferência sexual_. Se foi o caso, é OK sim. Mas acho que de repente o melhor recorte seria perguntar as duas coisas: _orientação sexual_ **e** _identidade de gênero_. Thaly, do @todasasletras fez esse [exemplo de formulário inclusivo](https://docs.google.com/forms/d/e/1FAIpQLSdR0x_dl6GW9D4xDo1euDyQzwlwzfciytf7n5pqTUbBWAkdhA/viewform) — ajuda será? Ou falei demaix e só compliquei…?

**Reply by anadulce** [reply]
Você foi perfeitamente claro, e já sanou minha dúvida! Muito obrigada!
Na verdade, eu é que falei de menos hahha Desculpa por não ter explicado bem o contexto.
Esse formulário de submissão de palestras vem sendo alterado ao longo dos últimos três ou quatro anos.
Esse "preferência sexual" foi adicionado em algum momento para substituir "orientação sexual" (tanto que já esteve inclusive na pergunta que se referia a orientação sexual, e nessa pergunta foi alterado outra vez, mas esquecemos de trocar no texto da apresentação do forms).
O motivo da minha pergunta é porque o motivo da troca para esse "preferência sexual" foi a alegação de que "orientação sexual" já não era aceitável e este seria o novo termo vigente, e não gostaríamos de substituir novamente algo que não era aceitável por outro termo igualmente errôneo, mas pela sua explicação, parece que não é o caso.
E ah, não é uma questão de confundir com identidade de gênero, pois há uma outra pergunta relativa à identidade de gênero no formulário e haverá uma análise correspondente nesse post, então dessa forma, creio que estamos seguindo a sua sugestão.
Infelizmente, esse formulário criado por Thaly chegou a nós depois que já havíamos aberto às submissões, mas realmente é uma excelente referência!
Muito obrigada por todas as explicações e pelas referências, e espero que a minha dúvida tenha ficado mais clara e mais bem embasada agora.
